Makefile

```diff
@@ -137,12 +141,12 @@ $(description): version/description
 	sed "s#HANDLER_IMAGE#$(HANDLER_IMAGE)#" \
 		version/description > $@
 
-prepare-patch:
-	./hack/prepare-release.sh patch
-prepare-minor:
-	./hack/prepare-release.sh minor
-prepare-major:
-	./hack/prepare-release.sh major
+prepare-patch: $(RELEASE_NOTES)
+	export PATH=$(GOBIN):$$PATH && ./hack/prepare-release.sh patch
+prepare-minor: $(RELEASE_NOTES)
+	export PATH=$(GOBIN):$$PATH && ./hack/prepare-release.sh minor
+prepare-major: $(RELEASE_NOTES)
+	export PATH=$(GOBIN):$$PATH && ./hack/prepare-release.sh major
```

Comment:
Please pass path to the binary as a standalone variable. The same way we do it with GITHUB_RELEASE in the release target.

Reply:
Maybe what we want is to pass GOBIN at top of Makefile so all of our scripts just use the executable and they don't depend on external variables and can be run locally if proper tools are there, what do you think ?

Reply:
Nah let's do as you say and do a follow up with PATH.